File: ProjetoPadraoFrontAngular/ProjetoPadraoFront/src/objects/Usuario.ts
import { EPerfil } from "src/enums/EPerfil";
import { RetornoPadrao } from "./RetornoPadrao";

export interface Usuario extends RetornoPadrao
{
    data: DataUsuario
}

export interface DataUsuario
{
    idUsuario: string, 
    nome: string,
    telefone: string,
    login: string,
    senha: string,
    cpf: string,
    perfil: EPerfil,
    idUbs: string
}

File: ProjetoPadraoDotnetCore/Infraestrutura.Data/Entity/Usuario.cs
namespace Infraestrutura.Entity;

public class Usuario
{
    public int IdUsuario { get; set; }
    public string Nome { get; set; }
    public string Email { get; set; }
    public string Cpf { get; set; }
    public string? Telefone { get; set; }
    public string Senha { get; set; }
}